test/integration/consul-container/libs/node/consul-container.go

```diff
@@ -26,6 +26,24 @@ type consulContainerNode struct {
 	container testcontainers.Container
```

Comment:
These changes are picking up some other changes that had not been backported from enterprise.